pages/src/main/java/uk/gov/android/ui/pages/AlertPage.kt

```diff
@@ -0,0 +1,165 @@
+import uk.gov.android.ui.components.buttons.ButtonType
+import uk.gov.android.ui.components.buttons.GdsButton
+import uk.gov.android.ui.theme.GdsTheme
+import uk.gov.android.ui.theme.mediumPadding
+import uk.gov.android.ui.theme.smallPadding
+
+@Suppress("LongMethod")
+@Composable
+fun AlertPage(
+    alertPageParameters: AlertPageParameters
+) {
+    alertPageParameters.apply {
+        GdsTheme {
+            Column(
+                modifier = Modifier.fillMaxSize(),
+                verticalArrangement = Arrangement.SpaceBetween
+            ) {
+                Column {
+                    Row(modifier = Modifier.padding(bottom = 12.dp)) {
+                        IconButton(onClick = onClose) {
+                            Icon(
+                                imageVector = Icons.Filled.Close,
+                                contentDescription = stringResource(
+                                    R.string.preview__alertPage__close
+                                ),
+                                tint = colors.primary
+                            )
+                        }
+                    }
+                    Row {
+                        GdsHeading(
+                            headingParameters = HeadingParameters(
+                                modifier = Modifier
+                                    .fillMaxWidth()
+                                    .padding(horizontal = smallPadding)
+                                    .padding(top = smallPadding),
+                                size = HeadingSize.H1(),
+                                text = title
+                            )
+                        )
+                    }
+                    Row {
+                        Text(
+                            color = colors.contentColorFor(colors.background),
+                            modifier = Modifier
+                                .fillMaxWidth()
+                                .padding(horizontal = smallPadding),
+                            style = MaterialTheme.typography.subtitle1,
+                            text = annotatedContent
+                        )
+                    }
+                }
+                Row {
+                    GdsButton(
+                        buttonParameters = ButtonParameters(
+                            buttonType = ButtonType.DANGER(),
+                            text = ctaText,
+                            modifier = Modifier
+                                .fillMaxWidth()
+                                .padding(
+                                    horizontal = smallPadding,
+                                    vertical = mediumPadding
+                                ),
+                            onClick = onPrimary
+                        )
+                    )
+                }
+            }
+        }
+    }
+}
+
+data class AlertPageParameters(
+    val title: Int,
+    val annotatedContent: AnnotatedString,
```

Comment:
This is ok, I don't necessarily mind this approach, gives more flexibility to the user of this page. We do already have a BulletList component though that brings it a little more inline with the style of other pages... up to you